src/features/session_trace/instrument/index.js

```diff
@@ -71,57 +58,23 @@ export class Instrument extends InstrumentBase {
       handle('bstHist', [location.pathname + location.hash, this.startPath, this.time], undefined, FEATURE_NAMES.sessionTrace, thisInstrumentEE)
     })
 
-    if (supportsPerformanceObserver()) {
-      // capture initial resources, in case our observer missed anything
-      handle(BST_RESOURCE, [window.performance.getEntriesByType('resource')], undefined, FEATURE_NAMES.sessionTrace, thisInstrumentEE)
```

Comment:
`[window.performance.getEntriesByType('resource')]` is older and more universally supported than the performance observer callback subscription.  For example, IE supports getEntriesByType, but not PerformanceObserver

Reply:
i guess that wasn't running anyway, was it?

Reply:
I think we might be able to find some reductions of our bundle sizes if were were to 1. pass the agent into the instrument file when instantiating and 2. pass the instrument into the aggregator when instantiating. Just in general, we have a lot of code dedicated to getting the correct event emitter, passing around the agent identifier, etc. Realistically, if the Agent class is instantiated for every agent on the page and manages all the instances of "things" passed down into features, we could probably strip out some code in the agent.